admin-tools/lib/src/main/scala/com/gu/mediaservice/ImagesBatchProjection.scala

```diff
@@ -0,0 +1,40 @@
+package com.gu.mediaservice
+
+import com.gu.mediaservice.lib.config.{ServiceHosts, Services}
+import com.gu.mediaservice.model.Image
+
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, ExecutionContext, Future}
+
+object ImagesBatchProjection {
+  def apply(apiKey: String, domainRoot: String, timeout: Duration): ImagesBatchProjection =
+    new ImagesBatchProjection(apiKey, domainRoot, timeout)
+}
+
+class ImagesBatchProjection(apiKey: String, domainRoot: String, timeout: Duration) {
+
+  private val ImageProjector = createImageProjector
+
+  def getMaybeImagesProjectionBlobs(mediaIds: List[String])(implicit ec: ExecutionContext): List[Either[Image, String]] = {
```

Comment:
Would `getImageProjections` be more intention revealing here?

Reply:
or `getImagesProjection` ?

Reply:
done